All_years/04-Social_Care/04-SDS_data.R

```diff
@@ -68,8 +68,8 @@ sds_full_clean <- sds_full_data %>%
   )) %>%
   # Fix sds_end_date is earlier than sds_start_date by setting end_date to be the end of fyear
   mutate(sds_end_date = if_else(
-    sds_start_date>=sds_end_date,
-    end_fy(year = period, 'alternate'),
+    sds_start_date >= sds_end_date,
```

Comment:
Just wondering of using `>=` here or `>` ? As in #430, we chose to use `>`, ie `service_start_date > service_end_date`,

Reply:
Yep, should be `>`, although I'm not really sure why this code change is on this branch at all 😂

Reply:
Might be my fault